src/FieldDescription/BaseFieldDescription.php

```diff
@@ -270,11 +268,12 @@ public function setOptions(array $options)
             unset($options['help']);
         }
 
-        // set default placeholder
+        // NEXT_MAJOR: Remove this.
         if (!isset($options['placeholder'])) {
```

Comment:
if not set, it will set a default, but if it is set to null, it won't, I think the new behavior you implemented don't cover this case

Reply:
isset(null) is false.